pkg/csi/service/wcp/controller.go

```diff
@@ -376,8 +378,14 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		case common.AttributeStoragePool:
 			storagePool = req.Parameters[paramName]
 		case common.AttributeStorageTopologyType:
-			// TODO: TKGS-HA : Add validation
+			// TKGS-HA: validate storageTopologyType.
 			storageTopologyType = req.Parameters[paramName]
+			val := strings.ToLower(storageTopologyType)
+			if val != "zonal" && val != "crosszonal" {
```

Comment:
remove `&& val != "crosszonal"` from this condition.

Reply:
Done